prow/config/config.go

```diff
@@ -111,6 +111,15 @@ type ProwConfig struct {
 
 	// GitHubOptions allows users to control how prow applications display GitHub website links.
 	GitHubOptions GitHubOptions `json:"github,omitempty"`
+
+	// StatusErrorLinkFromConfig is the url that will be used for jenkins prowJobs that can't be
+	// found, or have another generic issue. The default that will be used if this is not set
+	// is: https://github.com/kubernetes/test-infra/issues
+	StatusErrorLinkString string `json:"status_error_link,omitempty"`
+
+	// StatusErrorLink is the url representation of StatusErrorLinkFromConfig. This
+	// variable should be used in all places internally.
+	StatusErrorLink *url.URL `json:"-"`
```

Comment:
Is this ever used?

Reply:
Seems like we can just make this a string and get rid of the duplicated StatusErrorLinkString

Reply:
Sure, I'm down with that. Then we will just disgard the url.URL at parsing?

Reply:
We don't need to parse anything. We never use what we parse anyway.

User configures StatusErrorLinkString, we set the status URL to StatusErrorLinkString

The stuff we do with StatusErrorLink is never used

Reply:
Makes sense... I'll delete `StatusErrorLink` and rename `StatusErrorLinkString` to ``StatusErrorLink`. I think we still want to verify that the url is valid though, right? The easiest way to do that is the use url.Parse() and throw away the returned *url.URL.

Is that what you would like?

Reply:
Nope, I'll delete it.

Reply:
Yeah, doing the parse on load and throwing it away sounds like the plan